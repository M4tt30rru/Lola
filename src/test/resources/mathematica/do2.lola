##Find do[##Any(_e),{##Identifier(_i),##Literal(_min),##Literal(_max),##Literal(_step)}];
 ##replace for(int ##(_i)=##(_min);##(_i)<=##(_max);##(_i)+=##(_step)) {##(_e);}

do[f(i);g(i * 2),{i,10,100,5}];
 