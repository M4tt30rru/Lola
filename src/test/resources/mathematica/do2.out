for(int i=10;i<=100;i+=5) {f(i);g(i * 2);}